appveyor.yml

```diff
@@ -31,6 +31,7 @@ test:
     - '**\Sarif.FunctionalTests.dll'
     - '**\Sarif.ValidationTests.dll'
     - '**\Sarif.Viewer.VisualStudio.UnitTests.dll'
+    - '**\Microsoft.CodeAnalysis.SarifCli.FunctionalTests.dll'
```

Comment:
Fixed.

---
In reply to: [76704201](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r76704201) [](ancestors = 76704201)